.rubocop_todo.yml

```diff
@@ -9,20 +9,26 @@
 # Offense count: 1
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Max: 42
+  Max: 28
```

Comment:
Wonder why this changed. Max block length is now 28? 🤔

Reply:
could it be the changes related to availability maybe 🤔